sdk/meta/api.go

```diff
@@ -333,10 +333,8 @@ create_dentry:
 		}
 		return nil, statusToErrno(status)
 	} else if status != statusOK {
-		if status != statusExist {
-			mw.iunlink(mp, info.Inode, mw.Client.GetLatestVer(), 0, fullPath)
-			mw.ievict(mp, info.Inode, fullPath)
-		}
+		mw.iunlink(mp, info.Inode, mw.Client.GetLatestVer(), 0, fullPath)
```

Comment:
Don't we need to judge this status code? If the dentry already exists when it is being created, the inode needs to be deleted because the file already exists. However, in other cases, it might be due to a network timeout. At this time, if the inode is deleted, the dentry might still remain.